docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/pdq/node--pdq-cancer-information-summary.html.twig

```diff
@@ -0,0 +1,35 @@
+{{ attach_library('cgov_common/pdq') }}
+{% import '@cgov_common/content_page_macros.twig' as contentPageMacros %}
+<!-- ********************************* BEGIN Page Content ********************************** -->
+
+<article>
+{# {{kint(_context)}} #}
+    <div class="resize-content">
+        <h1>{{- node.label -}}</h1>
+        {{ drupal_block('page_options') }}
+        <div id="cgvBody">
+            <div class="summary-sections">
+                {% if content.field_summary_sections|length > 0 %}
+                <nav class="on-this-page" role="navigation">
+                    <h6>{% trans %}On This Page{% endtrans %}</h6>
```

Comment:
Added `trans` twig block, but it does not seem to work. No translation on Spanish pages. Seeing the same for Article Pages where I grabbed the code from.